lib/livebook/hubs/team.ex

```diff
@@ -67,18 +69,14 @@ defimpl Livebook.Hubs.Provider, for: Livebook.Hubs.Team do
   alias Livebook.Teams
 
   def load(team, fields) do
-    %{
-      team
-      | id: fields.id,
-        session_token: fields.session_token,
-        teams_key: fields.teams_key,
-        org_public_key: fields.org_public_key,
-        org_id: fields.org_id,
-        user_id: fields.user_id,
-        org_key_id: fields.org_key_id,
-        hub_name: fields.hub_name,
-        hub_emoji: fields.hub_emoji
-    }
+    for {key, value} <- fields, reduce: team do
+      acc ->
+        if is_map_key(team, key) do
+          Map.replace!(acc, key, value)
+        else
+          acc
+        end
+    end
```

Comment:
Why do we need this change?

Reply:
This will make the hub loading more dynamic without crashing if we add or remove a field

Reply:
I think you can write this as `struct(team, fields)`.